homeassistant/components/jvc_projector/config_flow.py

```diff
@@ -56,19 +58,23 @@ async def async_step_user(
                 return self.async_create_entry(
                     title=NAME,
                     data={
+                        CONF_NAME: name,
                         CONF_HOST: host,
                         CONF_PORT: port,
                         CONF_PASSWORD: password,
+                        CONF_READONLY: readonly,
                     },
                 )
 
         return self.async_show_form(
             step_id="user",
             data_schema=vol.Schema(
                 {
+                    vol.Required(CONF_NAME): str,
```

Comment:
Users can already rename the config entry or device. We shouldn't add name here.